### ISSUE TITLE
[bug] Forward slashes are not escaped

### ISSUE THREAD

**Issue by Timic3** [soketi/pws#21]
It looks like forward slashes are not escaped when passing array via `broadcastWith` function.

The following works:
```php
public function broadcastWith()
{
    return ['a' => 'abc']; // client receives this correctly
}
```
Received payload: `{"event":"updated","channel":"resources","data":"{\"a\":\"abc\"}"}`

while this does not:
```php
public function broadcastWith()
{
    return ['a' => 'abc/d']; // does not even send it to the client
}
```
No payload received.

Archived echo-server escapes it correctly:
`42/echo-key,["updated","resources","{\"a\":\"abc\\/d\"}"]`

Laravel and WebSocket (pWS) server return no errors.

**Comment by rennokki** [comment]
I have encountered this issue back when I wrote Echo Server, and it was because of the fact that the received data that eventually gets into the hash function to check the secret is already encoded, so I have to look at this.

**Comment by rennokki** [comment]
Released 0.2.0 with the appropriate fixes
